packages/react/src/progress-indicator/ProgressIndicator.stories.tsx

```diff
@@ -31,12 +31,28 @@ const exampleButtonItems: ProgressIndicatorItem[] = [
 	{ label: 'Review and submit', status: 'blocked', onClick: console.log },
 ];
 
+const exampleCustomActiveItem: ProgressIndicatorItem[] = [
+	{ label: 'Introduction', status: 'done', href: '#', isActive: true },
+	{ label: 'Submit evidence', status: 'saved', href: '#' },
+	{ label: 'Organisations', status: 'started', href: '#' },
+	{ label: 'Business contacts', status: 'error', href: '#' },
+	{ label: 'Case studies', status: 'todo', href: '#' },
+	{ label: 'Attachments', status: 'doing', href: '#' },
+	{ label: 'Review and submit', status: 'blocked', href: '#' },
+];
+
 export const Basic: Story = {
 	args: {
 		items: exampleLinkItems,
 	},
 };
 
+export const CustomActiveItem: Story = {
```

Comment:
Probably shouldn't be "custom" anymore

Reply:
I've removed that export completly and updated the other examples to the new format. We don't want any of our documentation promoting the old "doing" status.